integrations/gateway-api-inference-extension/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+sample_dashboards:
+  -
+    category: Inference Extension
+    id: gateway-api-inference-extension
+    display_name: Inference Extension Prometheus Overview
+    description: "This dashboard has charts displaying throughput, latency, cache usage, and other metrics for inference extension."
```

Comment:
Should inference extension here be capitalized? "and other metrics for Inference Extension"

Reply:
Done.